src/uproot/interpretation/library.py

```diff
@@ -611,18 +608,13 @@ def group(self, arrays, expression_context, how):
         elif how is list:
             return [arrays[name] for name, _ in expression_context]
         elif how is dict:
-            return dict(
-                (_rename(name, c), arrays[name]) for name, c in expression_context
-            )
+            return {_rename(name, c): arrays[name] for name, c in expression_context}
```

Comment:
This was available in 2.7+, IIRC.

Reply:
But not in 2.6. That's also why the format strings didn't have any `{}` in them (also a 2.7+ feature). None of that matters now, though.